### PR TITLE
Update developer to community

### PR DESCRIPTION
**Pull request by ronykoz**
## Status
- [x] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
Update developer contribution header to community 

image

### CODE REVIEW

File: demisto_sdk/commands/unify/unifier.py [hunk 1]
```diff
@@ -23,7 +23,7 @@
 
 PACK_METADATA_PATH = 'pack_metadata.json'
 CONTRIBUTOR_DISPLAY_NAME = ' ({} contribution)'
-CONTRIBUTOR_DETAILED_DESC = '### This is a {} contributed integration' \
+CONTRIBUTOR_DETAILED_DESC = '### {} contributed integration' \
```
Comment (glicht) [inline demisto_sdk/commands/unify/unifier.py:26]: Let's capitalize this
```suggestion
CONTRIBUTOR_DETAILED_DESC = '### {} Contributed Integration' \
```

File: demisto_sdk/commands/unify/unifier.py [hunk 2]
```diff
@@ -23,7 +23,7 @@
 
 PACK_METADATA_PATH = 'pack_metadata.json'
 CONTRIBUTOR_DISPLAY_NAME = ' ({} contribution)'
```
Comment (glicht) [inline demisto_sdk/commands/unify/unifier.py:25]: Let's capitalize
 ```suggestion
CONTRIBUTOR_DISPLAY_NAME = ' ({} Contribution)'
```

**Comment by ronykoz** [comment]
@glicht fixed as you requested, screenshot attached at the main comment

File: demisto_sdk/commands/unify/unifier.py [hunk 3]
```diff
@@ -22,9 +22,12 @@
 from ruamel.yaml.scalarstring import FoldedScalarString
 
 PACK_METADATA_PATH = 'pack_metadata.json'
-CONTRIBUTOR_DISPLAY_NAME = ' ({} contribution)'
-CONTRIBUTOR_DETAILED_DESC = '### This is a {} contributed integration' \
-                            '\nFor all questions and enhancement requests please contact the partner directly:'
+CONTRIBUTOR_DISPLAY_NAME = ' ({} Contribution)'
+CONTRIBUTOR_DETAILED_DESC = '### {} Contributed integration\n' \
```
Comment (glicht) [inline demisto_sdk/commands/unify/unifier.py:26]:
```suggestion
CONTRIBUTOR_DETAILED_DESC = '### {} Contributed Integration\n' \
```

File: demisto_sdk/commands/unify/unifier.py [hunk 4]
```diff
@@ -22,9 +22,12 @@
...
+                            '#### Integration Author: {}\n' \
+                            'Support and maintenance for this integration is provided by the author. ' \
+                            'Please use the following details to contact:'
```
Comment (glicht) [inline demisto_sdk/commands/unify/unifier.py:29]:
```suggestion
                            'Please use the following contact details:'
```

File: demisto_sdk/commands/unify/unifier.py [hunk 5]
```diff
@@ -382,23 +386,24 @@ def get_contributor_data(self):
...
         Returns:
             The unified yaml file.
         """
         unified_yml['display'] += CONTRIBUTOR_DISPLAY_NAME.format(contributor_type.capitalize())
         existing_detailed_description = unified_yml.get('detaileddescription', '')
-        contributor_description = CONTRIBUTOR_DETAILED_DESC.format(contributor_type)
+        contributor_description = CONTRIBUTOR_DETAILED_DESC.format(contributor_type.capitalize(), author)
         if contributor_email:
-            contributor_description += f'\n**Email** - [mailto](mailto:{contributor_email})'
+            contributor_description += f'\n**Email** - [{contributor_email}](mailto:{contributor_email})'
```
Comment (glicht) [inline demisto_sdk/commands/unify/unifier.py:406]: Let's do a bullet
```suggestion
            contributor_description += f'\n- **Email**: [{contributor_email}](mailto:{contributor_email})'
```

File: demisto_sdk/commands/unify/unifier.py [hunk 6]
```diff
@@ -382,23 +386,24 @@ def get_contributor_data(self):
...
         if contributor_url:
             contributor_description += f'\n**URL** - [{contributor_url}]({contributor_url})'
```
Comment (glicht) [inline demisto_sdk/commands/unify/unifier.py:408]: Let's do a bullet
```suggestion
            contributor_description += f'\n- **URL**: [{contributor_url}]({contributor_url})'
```

File: demisto_sdk/commands/unify/unifier.py [hunk 7]
```diff
@@ -22,9 +22,12 @@
 from ruamel.yaml.scalarstring import FoldedScalarString
 
 PACK_METADATA_PATH = 'pack_metadata.json'
-CONTRIBUTOR_DISPLAY_NAME = ' ({} contribution)'
-CONTRIBUTOR_DETAILED_DESC = '### This is a {} contributed integration' \
-                            '\nFor all questions and enhancement requests please contact the partner directly:'
+CONTRIBUTOR_DISPLAY_NAME = ' ({} Contribution)'
+CONTRIBUTOR_DETAILED_DESC = '### {} Contributed integration\n' \
+                            '#### Integration Author: {}\n' \
+                            'Support and maintenance for this integration is provided by the author. ' \
```
Comment (glicht) [inline demisto_sdk/commands/unify/unifier.py:28]:
```suggestion
                            'Support and maintenance for this integration are provided by the author. ' \
```

**Pull request merged**